win32ss/gdi/gdi32/objects/dc.c

```diff
@@ -1060,6 +1060,13 @@ SetBkMode(
     PDC_ATTR pdcattr;
     INT iOldMode;
 
+    /* Avoid bad mode setting */
+    if((iBkMode != TRANSPARENT) & (iBkMode != TRANSPARENT))
```

Comment:
Why are you checking the exact same condition twice? I suspect that you intended instead:
```
    if ((iBkMode != TRANSPARENT) & (iBkMode != OPAQUE))
```
Also, note <space> after the ```if```.